dependency-graph/properties/dependency-review.properties.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Dependency Review",
+    "creator": "GitHub",
+    "description": "Scans Pull Requests on each push for the introduction and/or resolution of vulnerable depdendencies to the repository",
+    "iconName": "octicon mark-github",
+    "categories": [
+      "Dependency Graph",
+      "Dependency Review",
+      "Go",
+      "Java",
+      "JavaScript",
+      "TypeScript",
+      "Python",
+      "Ruby",
+      "Actions",
```

Comment:
Why `Actions`?

Reply:
We pick up Actions dependencies declared on repos that include a `.github/workflows` dir and detect vulnerable versions, same as with other language ecosystem packages